frontend/src/pages/AddWebsitePage.vue

```diff
@@ -0,0 +1,14 @@
+<template>
+    <AddWebsiteMaster />
```

Comment:
it's probably doesn't make sense to create such page, since `AddWebsiteMaster` can be used as page (I mean move all from AddWebsiteMaster here and remove this `AddWebsitePage.vue`)